### PR TITLE
Add the searching functionality

### PR DESCRIPTION
**Pull request by LucasBoberg**
This pr adds the searching functionality to the code, it works by providing a pattern and a file path as command arguments

### CODE REVIEW

File: src/main/java/se/lucasboberg/Main.java [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+            return;
+        }
+        var pattern = args[0];
+        var path = args[1];
+
+        try {
+            var finder = new Finder(pattern, path);
+            finder.find();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```
Comment (gabrielsjogren) [inline src/main/java/se/lucasboberg/Main.java]: Maybe you can handle this a little bit better. At least add logging.

File: src/main/java/se/lucasboberg/Finder.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    public void find() throws IOException {
+        List<Tuple<Integer, String>> lines = readLines();
+        for (Tuple<Integer, String> line : lines) {
+            if (pattern.matcher(line.getSecond()).find()) {
+                System.out.println("line " + line.getFirst() + ": " + line.getSecond());
+            }
+        }
+    }
+
+    private List<Tuple<Integer, String>> readLines() throws IOException {
```
Comment (gabrielsjogren) [inline src/main/java/se/lucasboberg/Finder.java:26]: Veri nice with private method. Good job

**Review by gabrielsjogren — CHANGES_REQUESTED** [review]

File: src/main/java/se/lucasboberg/Main.java (hunk 1 above)
Comment (LucasBoberg) [reply src/main/java/se/lucasboberg/Main.java]: Fixed!

**Review by gabrielsjogren — APPROVED** [review]
Really good error handling now. thanks!

**Pull request merged**